cli/src/program_v4.rs

```diff
@@ -278,7 +278,6 @@ impl ProgramV4SubCommands for App<'_, '_> {
                             Arg::with_name("all")
                                 .long("all")
                                 .conflicts_with("account")
-                                .conflicts_with("buffer_authority")
```

Comment:
@pgarg66 it's supposed to be `.conflicts_with("authority")`, right?

Reply:
Shall I push up that change so we can merge this? May as well fix this

Reply:
That's correct. It should've been the `authority`. Likely a copy-paste error from `program.rs`.
Thanks for catching it.

Reply:
Pushed the change to this PR.